frontend/src/action-creator.sync.js

```diff
@@ -118,35 +113,36 @@ export const updateWorkspaceDetail = workspaceDetail => ({ type: `${UPDATE}/${WO
 export const WORKSPACE_LOADED = `${WORKSPACE}/Loaded`
 export const setWorkspaceLoaded = () => ({ type: `${SET}/${WORKSPACE_LOADED}` })
 
-const MEMBER = 'Member'
-export const WORKSPACE_MEMBER = `${WORKSPACE}/${MEMBER}`
-export const WORKSPACE_MEMBER_LIST = `${WORKSPACE_MEMBER}/List`
-export const setWorkspaceMemberList = workspaceMemberList => ({ type: `${SET}/${WORKSPACE_MEMBER_LIST}`, workspaceMemberList })
-// INFO - CH - 2023-11-22 - WORKSPACE_MEMBER_ADD is used by api function postWorkspaceMember()
+const ROLE = 'Role'
+export const USER_ROLE = `${USER}/${ROLE}`
+export const USER_ROLE_LIST = `${USER_ROLE}/List`
+export const setWorkspaceMemberList = workspaceMemberList => ({ type: `${SET}/${USER_ROLE_LIST}`, workspaceMemberList })
```

Comment:
rename setWorkspaceMemberList to setUserRoleList = userRoleList =>...
and create a const for `${SET}/${USER_ROLE_LIST}` and use it in reducers

Reply:
done

Reply:
the const ${SET}/${USER_ROLE_LIST} hasn't been created